src/main/harmonysolver.qml

```diff
@@ -526,10 +529,7 @@ MuseScore {
                         )
             }
             if(cursor.element.notes[0].tieForward !== null){
-                  throw new Errors.FiguredBassInputError(
```

Comment:
Proponuję tu zostawić informację, gdzie występuje tie

Reply:
Bo jak wystąpi więcej razy, to dostajemy warninga z tą samą informacją powtórzoną wielokrotnie

Reply:
Hmm, to mogę zrobić jeszcze jedną zmienną, w której będą wszystkie takty z ligaturą, bo ich potencjalnie może być więcej i żeby nie doklejać kilka razy tego samego

Reply:
Faktycznie, przeoczyłem to :)

Reply:
![image](https://user-images.githubusercontent.com/48293681/101963552-47b2ac80-3c0f-11eb-8ff7-caee4d97612f.png)